### PR TITLE
Create default files

### PR DESCRIPTION
**Pull request by chrisgarrity**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+### Ways to Help
+
+* **Documenting bugs**
+  * If you've identified a bug in Scratch or ScratchJr you should first check to see if it's been filed as an issue, if not you can file one.  Make sure you follow the issue template.
+  * It's important that we can consistently reproduce issues. When writing an issue, be sure to follow our [reproduction step guidelines](https://github.com/LLK/scratch-gui/wiki/Writing-good-repro-steps).
+    * Some issues are marked "Needs Repro". Adding a comment with good reproduction steps to those issues is a great way to help.
+  * If you don't have an issue in mind already, you can look through the [Bugs & Glitches forum.](https://scratch.mit.edu/discuss/3/) Look for users reporting problems, reproduce the problem yourself, and file new issues following our guidelines. 
+
+* **Fixing bugs**
+  * You can request to fix a bug in a comment on the issue if you at mention the repo coordinator, who for this repo is @chrisgarrity.
```
Comment (cwillisf) [inline CONTRIBUTING.md]: should refer or even link to "Repository Coordinators" section below

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+* scratch-vm @ericrosenbaum
+* scratch-www @benjiwheeler
+* scratchjr @chrisgarrity
+* scratchjr-website @chrisgarrity
+
+Repositories that do not accept external PRs:
+* scratch-parser
+* scratch-render-fonts
+* scratch-resources
+* scratch-sb1converter
```
Comment (cwillisf) [inline CONTRIBUTING.md:44]: add scratch-l10n

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+**Important:** we follow the [Github Flow process](https://guides.github.com/introduction/flow/) as our development process.
+
+### How to Fix Bugs
+1. Identify which Github issue you are working on. Leave a comment on the issue to let us (and other contributors) know you're working on it.
+2. Make sure you have a fork of this repo (see [Github's forking a repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) for details)
+3. Switch to the `develop` branch, and pull down the latest changes from upstream
+4. Run the code, and reproduce the problem
+5. Create your branch from the `develop` branch
+6. Make code changes to fix the problem
+7. Run `npm test` to make sure that your changes pass our tests
```
Comment (cwillisf) [inline CONTRIBUTING.md]: `npm test` might not be the right command in all repositories. We should probably make it so it is, though...

Comment (cwillisf) [reply CONTRIBUTING.md]: I checked all of the repositories I've cloned on my computer and "npm test" is right for all of them, so my worry might be unfounded :)

Comment (chrisgarrity) [reply CONTRIBUTING.md]: I updated to clarify that `npm test` is often the command, but the main thing is that they should test their changes.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+### Ways to Help
+
+* **Documenting bugs**
+  * If you've identified a bug in Scratch or ScratchJr you should first check to see if it's been filed as an issue, if not you can file one.  Make sure you follow the issue template.
+  * It's important that we can consistently reproduce issues. When writing an issue, be sure to follow our [reproduction step guidelines](https://github.com/LLK/scratch-gui/wiki/Writing-good-repro-steps).
+    * Some issues are marked "Needs Repro". Adding a comment with good reproduction steps to those issues is a great way to help.
+  * If you don't have an issue in mind already, you can look through the [Bugs & Glitches forum.](https://scratch.mit.edu/discuss/3/) Look for users reporting problems, reproduce the problem yourself, and file new issues following our guidelines. 
+
+* **Fixing bugs**
+  * You can request to fix a bug in a comment on the issue if you at mention the repo coordinator for this repository. Check the list of [Repository Coordinators](./CONTRIBUTING.md#repository-coordinators) to find out who to @-mention.
```
Comment (benjiwheeler) [inline CONTRIBUTING.md]:
```suggestion
  * You can request to fix a bug in a comment on the issue if you mention the repo coordinator for this repository, using the @ symbol with their Github username. Check the list of [Repository Coordinators](./CONTRIBUTING.md#repository-coordinators) to find out who to @-mention.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
+## Contributing
+The development of Scratch is an ongoing process, and we love to have people in the Scratch, ScratchJr, and open source communities help us along the way.
+
+### Ways to Help
+
+* **Documenting bugs**
+  * If you've identified a bug in Scratch or ScratchJr you should first check to see if it's been filed as an issue, if not you can file one.  Make sure you follow the issue template.
```
Comment (benjiwheeler) [inline CONTRIBUTING.md]:
```suggestion
  * If you've identified a bug in Scratch or ScratchJr, you should first check to see if it's been filed as an issue. If it has not been filed, please file an issue!  Make sure you follow the issue template.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  * It's important that we can consistently reproduce issues. When writing an issue, be sure to follow our [reproduction step guidelines](https://github.com/LLK/scratch-gui/wiki/Writing-good-repro-steps).
+    * Some issues are marked "Needs Repro". Adding a comment with good reproduction steps to those issues is a great way to help.
+  * If you don't have an issue in mind already, you can look through the [Bugs & Glitches forum.](https://scratch.mit.edu/discuss/3/) Look for users reporting problems, reproduce the problem yourself, and file new issues following our guidelines. 
+
+* **Fixing bugs**
+  * You can request to fix a bug in a comment on the issue if you at mention the repo coordinator for this repository. Check the list of [Repository Coordinators](./CONTRIBUTING.md#repository-coordinators) to find out who to @-mention.
+    * If the issue is marked "Help Wanted" you can go ahead and start working on it!
+  * **We will only accept Pull Requests for bugs that have an issue filed that has a priority label** 
+    * If you're interested in fixing a bug with no issue, file the issue first and wait for it to have a priority added to it.
```
Comment (benjiwheeler) [inline CONTRIBUTING.md]:
```suggestion
    * If you're interested in fixing a bug that does not have an issue already, please file the issue first and wait for it to have a priority added to it.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  
+  * We are not looking for Pull Requests ("PR") for every issue and may deny a PR if it doesn't fit our criteria.
+    * We are far more likely to accept a PR if it is for an issue marked with Help Wanted.
```
Comment (benjiwheeler) [inline CONTRIBUTING.md]:
```suggestion
    * We are far more likely to accept a PR if it is for an issue marked with the "Help Wanted" label.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    * We will not accept PRs for issues marked with "Needs Discussion" or "Needs Design."
+    * Wait until an issue has been assigned a priority before working on it. Also leave a comment if you start working on a prioritized issue to avoid duplicating efforts.
```
Comment (benjiwheeler) [inline CONTRIBUTING.md]:
```suggestion
    * Wait until an issue has been assigned a priority before working on it. Also leave a comment if you start working on a prioritized issue, so other people don't start working on it also.
```

File: PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+_Describe what this Pull Request does_
+
+### Reason for Changes
+
+_Explain why these changes should be made. Be sure to include a brief summary of the issue the Pull Request solves too._
+
+### Test Coverage
+
+_Please show how you have added tests to cover your changes or provide step by step instructions for testing your changes. Include screenshots if applicable.
```
Comment (benjiwheeler) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
_Please show how you have added tests to cover your changes, or provide step-by-step instructions for testing your changes. Include screenshots if applicable. 
```

**Comment by benjiwheeler** [comment]
I made a few suggested edits for reading clarity and (hopefully) to make translating this with Google Translate more likely to be clear.(It was helpful to read this with fresh eyes, after being so familiar with it a while ago!).

Please feel free to accept those or ignore them, and otherwise LGTM!

**Pull request merged**